MicroverseTrials-main/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">Please complete...</p>
+
+    <form id="survey-form">
```

Comment:
To improve your styling approach, it's recommended to utilize classes instead of IDs. Unlike IDs, classes can be applied to multiple elements in a document, making them more adaptable and reusable. Utilizing classes can help save time and minimize the duplication of code.